index.html

```diff
@@ -940,8 +940,18 @@ <h2 class='informative'>Vary</h2>
       <section id="transmission-fixity" class="informative">
         <h2>Transmission Fixity</h2>
         <p>
-          Transmission fixity is verified by application of the <code>Digest</code> header
-          defined in [[RFC3230]] to <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+          Transmission fixity may be verified by including a <code>Digest</code> header (defined in
+          [[RFC3230]]) in <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+        </p>
+      </section>
+      <section id="persistence-fixity" class="informative">
+        <h2>Persistence Fixity</h2>
+        <p>
+          Persistence fixity may be verified by retrieving the current checksum of an <a>LDP-NR</a> and
```

Comment:
@escowles This doesn't address the nuance that `Want-Digest` is inherently about transmission, and drawing the conclusion that it is suitable for persistence fixity requires knowledge of the nature of the LDP-NR.  Can the editors comment about this nuance.  In particular, is the spec presuming that all LDP-NRs are persisted, and therefore by definition getting their checksum is measuring their persistence fixity?  If the spec is not making this presumption, then is the nuance too complicated/subtle to mention?